src/net.h

```diff
@@ -358,6 +358,19 @@ class CNode
 #endif
 
 public:
+    /** This node's max acceptable number ancestor transactions.  Used to decide whether this node will accept a
+     * particular transaction. */
+    size_t nLimitAncestorCount = BCH_DEFAULT_ANCESTOR_LIMIT;
```

Comment:
nit: same here the name of the variable could be confused for a limit applied to the BU node itself rather than one of its peers. (same for the other following 3 params)

Reply:
these are member variables... it would be redundant to further qualify them.